midend/copyStructures.h

```diff
@@ -53,19 +68,15 @@ namespace P4 {
  *
  */
 class DoCopyStructures : public Transform {
+    /// The type map.
     TypeMap *typeMap;
-    /// Specific targets may allow functions or methods to return structs.
-    /// Such methods will not be converted in this pass. Setting the
-    /// errorOnMethodCall flag will produce an error message if such a
-    ///  method is encountered.
-    bool errorOnMethodCall;
 
-    /// Do not only copy normal structures but also perform copy assignments for headers.
-    bool copyHeaders;
+    /// Configuration options.
+    CopyStructuresConfig _config;
```

Comment:
Any reason to use the underscore in name?

Reply:
It's a habit I have picked up recently. I can change it back. The reason it is useful is that you can declare private members as `_member` and then the public getter as `member()`.

Reply:
I can see why this is useful (I used to use this convention myself), but my point was more that it is quite unusual in P4C codebase.

Reply:
Yup, will change it back. We have this style in the testgen back end so sometimes it creeps back in. 

One problem is that the style in P4C can be random and changes from file to file.
Ideally, we should push https://github.com/p4lang/p4c/pull/4410 forward.